examples/nodejs/storage-load-gen/load-gen.ts

```diff
@@ -0,0 +1,287 @@
+of a CPU core then you most likely won't be able to improve throughput further
+without running additional nodejs processes.
+
+CPU will also impact your client-side latency; as you increase the number of
+concurrent requests, if they are competing for CPU time then the observed
+latency will increase.
+
+Also, since performance will be impacted by network latency, you'll get the best
+results if you run on a cloud VM in the same region as your Momento store.
+
+Check out the configuration settings at the bottom of the 'load-gen.ts' to
+see how different configurations impact performance.
+
+If you have questions or need help experimenting further, please reach out to us!
+`;
+
+async function main(loadGeneratorOptions: BasicLoadGenOptions) {
+  const loadGenerator = new BasicLoadGen(loadGeneratorOptions);
+  await loadGenerator.run();
+}
+
+const loadGeneratorOptions: BasicLoadGenOptions = {
+  /**
+   * This setting allows you to control the verbosity of the log output during
+   * the load generator run.
+   */
+  loggerFactory: new DefaultMomentoLoggerFactory(
+    /**
+     * Available log levels are trace, debug, info, warn, and error.
+     */
+    DefaultMomentoLoggerLevel.DEBUG
+  ),
+  /** Print some statistics about throughput and latency every time this many
+   *  seconds have passed.
+   */
+  showStatsIntervalSeconds: 5,
+  /**
+   * Configures the Momento client to timeout if a request exceeds this limit.
+   * Momento client default is 5 seconds.
+   */
+  requestTimeoutMs: 15 * 1000,
+  /**
+   * Controls the size of the payload that will be used for the store items in
+   * the load test.  Smaller payloads will generally provide lower latencies than
+   * larger payloads.
+   */
+  storeItemPayloadBytes: 100,
+  /**
+   * Sets an upper bound on how many requests per second will be sent to the server.
+   * Momento stores have a default throttling limit of 100 requests per second,
+   * so if you raise this, you may observe throttled requests.  Contact
+   * support@momentohq.com to inquire about raising your limits.
+   */
+  maxRequestsPerSecond: 100,
+  /**
+   * Controls the number of concurrent requests that will be made (via asynchronous
+   * function calls) by the load test.  Increasing this number may improve throughput,
+   * but it will also increase CPU consumption.  As CPU usage increases and there
+   * is more contention between the concurrent function calls, client-side latencies
+   * may increase.
+   *
+   * **Note**: You are likely to see degraded performance if you increase this above 50
+   * and observe elevated client-side latencies.
+   */
+  numberOfConcurrentRequests: 10,
+  /**
+   * Controls how long the load test will run, in milliseconds. We will execute operations
+   * for this long and the exit.
+   */
+  totalSecondsToRun: 60,
+  /**
+   * Controls whether the load generator will make GET, PUT, or both GET and PUT requests.
+   */
+  loadGenOperations: LoadGenOperations.GET_AND_PUT,
```

Comment:
added this to make my testing easier, but I can remove this stuff if it doesn't seem necessary or helpful

Reply:
i think this is fine!